integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIstioSessionMigrIstioEnabled.java

```diff
@@ -0,0 +1,402 @@
+        + "and session create time {2} before shutting down the primary server",
+        origPrimaryServerName, origSecondaryServerName, origSessionCreateTime);
+
+    // stop the primary server by changing ServerStartPolicy to NEVER and patching domain
+    logger.info("Shut down the primary server {0}", origPrimaryServerName);
+    shutdownServerUsingServerStartPolicy(origPrimaryServerName);
+
+    // send a HTTP request to get server and session info after shutting down the primary server
+    serverName = domainUid + "-" + origSecondaryServerName;
+    httpDataInfo = getServerAndSessionInfoAndVerify(serverName, webServiceGetUrl, " -b ");
+    // get server and session info from web service deployed on the cluster
+    String primaryServerName = httpDataInfo.get(primaryServerAttr);
+    String sessionCreateTime = httpDataInfo.get(sessionCreateTimeAttr);
+    String countStr = httpDataInfo.get(countAttr);
+    int count = Optional.ofNullable(countStr).map(Integer::valueOf).orElse(0);
+    logger.info("After patching the domain, the primary server changes to {0} "
+        + ", session create time {1} and session state {2}",
+        primaryServerName, sessionCreateTime, countStr);
+
+    // verify that a new primary server is picked and HTTP session state is migrated
+    assertAll("Check that WebLogic server and session vars is not null or empty",
+        () -> assertNotEquals(origPrimaryServerName, primaryServerName,
+            "After the primary server stopped, another server should become the new primary server"),
+        () -> assertEquals(origSessionCreateTime, sessionCreateTime,
+            "After the primary server stopped, HTTP session state should be migrated to the new primary server"),
+        () -> assertEquals(count, SESSION_STATE,
+            "After the primary server stopped, HTTP session state should be migrated to the new primary server")
+    );
+
+    finalPrimaryServerName = primaryServerName;
+
+    logger.info("Done testSessionMigration \nThe new primary server is {0}, it was {1}. "
+        + "\nThe session state was set to {2}, it is migrated to the new primary server.",
+        primaryServerName, origPrimaryServerName, SESSION_STATE);
+  }
+
+  private static int configIstioModelInImageDomain(String miiImage,
+                                                   String domainNamespace,
+                                                   String domainUid,
+                                                   String managedServerPrefix) {
+
+    // Create the repo secret to pull the image
+    // this secret is used only for non-kind cluster
+    createOcirRepoSecret(domainNamespace);
+
+    // create secret for admin credentials
+    logger.info("Create secret for admin credentials");
+    String adminSecretName = "weblogic-credentials";
+    assertDoesNotThrow(() -> createSecretWithUsernamePassword(
+        adminSecretName,
+        domainNamespace,
+        "weblogic",
```

Comment:
weblogic --> use the TestConstant.ADMIN_USERNAME_DEFAULT
welcome1 -->use the TestConstant.ADMIN_PASSWORD_DEFAULT